src/ui/components/SecondaryToolbox/ReactDevTools.tsx

```diff
@@ -137,6 +140,8 @@ export function ReactDevtoolsPanel() {
     replayClient
   );
 
+  const showRecordingTooLongWarning = !mightHaveRoutines;
```

Comment:
Whether a recording has routines or not isn't necessarily related to its length. I think I know why you're using this as a signal (it's all we have) but I'm concerned about false positives here.